modules/tensor_mechanics/src/materials/ComputeMultiPlasticityStress.C

```diff
@@ -77,6 +77,9 @@ ComputeMultiPlasticityStress::ComputeMultiPlasticityStress(const InputParameters
     _my_elasticity_tensor(RankFourTensor()),
     _my_strain_increment(RankTwoTensor())
 {
+  if (_epp_tol <= 0)
+    mooseError("ComputeMultiPlasticityStress: ep_plastic_tolerance must be positive");
+
```

Comment:
Yeah, you can call `setEpsilon(tol)` on an `FParser` object to set the tolerance for the fuzzy compare.